frontend/src/global_styles/content/_accounts.sass

```diff
@@ -119,3 +119,6 @@
 
   &--agreement
     font-size: inherit
+
+.login-auth-providers-title ~ .login-auth-provider-list
```

Comment:
Hi @aaron-contreras It correctly fix the bug, but I have an alternative way that help us with reducing custom css.

You can add primer class 'mb-3' to the element with 'login-auth-providers-title' class:

```
      <h3 class="login-auth-providers-title mb-3"><span>
          <%= local_assigns[:omniauth_title] || I18n.t('account.login_with_auth_provider')%>
        </span></h3>
```
It adds 1rem margin bottom to this element.
https://primer.style/foundations/css-utilities/margin

Reply:
Ah, great! I forget this CSS is available to use outside Primer view components. Thank you :)